internals/schemas/README.md

```diff
@@ -0,0 +1,10 @@
+# Configuration
```

Comment:
Why not place this file in the `docs` folder?

Reply:
Sorry, I thought this is what you meant with: `Might as well remove the entire file and replace it with docs/schemas/README.md?'

Reply:
So I just remove the file?